servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -74,9 +76,58 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
             from components.java
             artifact(javadocJar)
             artifact(sourcesJar)
+            versionMapping {
```

Comment:
This is a Gradle's incubating feature. We didn't use it before and let's remove it until necessary.